images-list

```diff
@@ -270,6 +271,7 @@ ghcr.io/epinio/epinio-unpacker rancher/mirrored-epinio-epinio-unpacker v1.6.2
 ghcr.io/epinio/epinio-unpacker rancher/mirrored-epinio-epinio-unpacker v1.8.1
 ghcr.io/epinio/epinio-unpacker rancher/mirrored-epinio-epinio-unpacker v1.9.0
 ghcr.io/epinio/epinio-unpacker rancher/mirrored-epinio-epinio-unpacker v1.10.0
+ghcr.io/jimmidyson/configmap-reload rancher/mirrored-jimmidyson-configmap-reload v0.13.1
```

Comment:
just to confirm, we are no longer sourcing this from `docker.io/jimmidyson/configmap-reload`

Reply:
Correct - same git repo source, but they stopped pushing to dockerhub and started using GHCR. I think in a scenario like that we can keep the same mirror destination right?

Note from upstream:
> Since version v0.10, the Docker image is available from ghcr.io at https://github.com/jimmidyson/configmap-reload/pkgs/container/configmap-reload. Previous versons are available from Docker Hub at https://hub.docker.com/r/jimmidyson/configmap-reload

Reply:
yeah sgtm